src/applications/vaos/actions/appointments.js

```diff
@@ -133,18 +133,17 @@ export function fetchFutureAppointments() {
               .add(13, 'months')
               .format('YYYY-MM-DD'),
           }),
-          getPendingAppointments(
-            moment()
+          getAppointmentRequests({
+            startDate: moment()
```

Comment:
I'm using the start and end dates that we had in place, but this made me do a double take since the range is the past 30 days.  @jbalboni do you remember if this queries the "created at" dates, or the option times?

Reply:
They're the created dates, I believe. The expectation is that these requests are resolved in a few days, so fetching 30 days of history should cover anything we want to show.